distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1985,40 +1770,48 @@ public void testCancelDownload() throws Exception {
                 RESOURCE_DOWNLOAD_ENDPOINT_ROOT.getUrl() + "?source=" + CSW_STUB_SOURCE_ID
                         + "&metacard=" + metacardId;
 
-        // @formatter:off
-        String downloadId = given().auth().preemptive().basic(LOCALHOST_USERNAME, LOCALHOST_PASSWORD)
-                .get(startDownloadUrl).then().log().all()
-                .extract().jsonPath().getString("downloadId");
-        // @formatter:on
+        given().auth()
+                .preemptive()
+                .basic(LOCALHOST_USERNAME, LOCALHOST_PASSWORD)
+                .get(startDownloadUrl)
+                .then()
+                .log()
+                .all();
 
-        localhostCometDClient.cancelDownload(downloadId);
+        expect("Waiting for download to start.").within(30, SECONDS)
+                .until(() -> localhostCometDClient.getDownloadIds()
+                        .size() > 0);
 
-        expect("Waiting for notifications.").within(10, SECONDS)
-                .until(() -> localhostCometDClient.getMessages(NOTIFICATIONS_CHANNEL)
-                        .size() == 1);
+        assertThat(localhostCometDClient.getDownloadIds()
```

Comment:
The main thread waits until the first id comes in, in CometDClient the first activities message appears (no other activity messages will be coming in besides the one we are triggering in the main thread. At this point, we have 1 download id, and this download id persists until we call, cancel the dowload. The download will not complete early because I made the ChunkedContent slow enough. If you are concerned the ChunkedContent is not slow enough, I can raise it back to 300 ms.